localsettings.example.py

```diff
@@ -87,22 +87,22 @@
 
 ####### # Email setup ########
 # email settings: these ones are the custom hq ones
-EMAIL_LOGIN = "notifications@dimagi.com"
+EMAIL_LOGIN = "notifications@yourdomain.com"
 EMAIL_PASSWORD = "******"
 EMAIL_SMTP_HOST = "smtp.gmail.com"
 EMAIL_SMTP_PORT = 587
 
 # Print emails to console so there is no danger of spamming, but you can still get registration URLs
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-ADMINS = (('HQ Dev Team', 'commcarehq-dev+www-notifications@dimagi.com'),)
-BUG_REPORT_RECIPIENTS = ['commcarehq-support@dimagi.com']
-NEW_DOMAIN_RECIPIENTS = ['commcarehq-dev+newdomain@dimagi.com']
-EXCHANGE_NOTIFICATION_RECIPIENTS = ['commcarehq-dev+exchange@dimagi.com']
+ADMINS = (('HQ Dev Team', 'commcarehq-dev+www-notifications@yourdomain.com'),)
+BUG_REPORT_RECIPIENTS = ['commcarehq-support@yourdomain.com']
+NEW_DOMAIN_RECIPIENTS = ['commcarehq-dev+newdomain@yourdomain.com']
+EXCHANGE_NOTIFICATION_RECIPIENTS = ['commcarehq-dev+exchange@yourdomain.com']
 
-SERVER_EMAIL = 'commcarehq-noreply@dimagi.com' #the physical server emailing - differentiate if needed
-DEFAULT_FROM_EMAIL = 'commcarehq-noreply@dimagi.com'
-SUPPORT_EMAIL = "commcarehq-support@dimagi.com"
+SERVER_EMAIL = 'commcarehq-noreply@yourdomain.com' #the physical server emailing - differentiate if needed
```

Comment:
E261 at least two spaces before inline comment
E262 inline comment should start with '# '